Dockerfile

```diff
@@ -256,7 +256,8 @@ COPY . /usr/local/src/ort
 WORKDIR /usr/local/src/ort
 
 # Prepare Gradle
-RUN scripts/import_proxy_certs.sh \
+RUN export GRADLE_USER_HOME=/usr/local/src/ort/ \
```

Comment:
It looks weird that this is pointing to ORT sources now. Shouldn't this rather be `/home/ort/.gradle`?

Reply:
> It looks weird that this is pointing to ORT sources now. Shouldn't this rather be `/home/ort/.gradle`?

This setting is only temporary. It is only in effect at build time of ORT. At build time the ort user is not used but root.
To my understanding this is the right place as the gradle wrapper uses the properties file that is copied in and this change is only making sure, that this properties file is expanded with the proxy settings. These settings will not be added to the final image as of now.

Reply:
But to my understanding it does not matter *when* this setting will be applied; setting Gradle's home directory to the ORT source code directory should *never* be correct.

Reply:
Ok, might be worth a discussion. But were does this leave us now?  
With the presented fix my problems are solved. I can build the ORT docker image behind a proxy again. I can also use the set_gradle_proxy.sh script in my workflows to append the proxy settings for gradle at runtime.

> it should be the [user-global properties file](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_configuration_properties) below the current user's home directory.

That is still possible but in the current setup of the dockerfile it does not at build time of the container.